packages/react-components/public/docs-root.css

```diff
@@ -0,0 +1,336 @@
+#docs-root .sbdocs-content {
```

Comment:
I don't see anything wrong with putting these styles in root, so we all our custom SB styles overrides in one place, and the per component SBs would also look consistent

Reply:
Is this because these styles were copied over from the website?

Reply:
I'll move these back to the root. I don't believe I'm doing anything in here that would be bad for the other SB instances, but I wanted to err on the side of caution.

Reply:
Got these moved over and it seems that React and web components use a completely different SB config. So seems safe to move these back to root for anyone that wants to inherit them.